README.md

```diff
@@ -50,7 +50,8 @@ Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Refe
 
   Key | Description | Default
   :--|:--|:--
-  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#specify-a-client-id-when-loading-the-api) | `undefined`
+  `channel` | [Client usage reporting channel](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
+  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
```

Comment:
This line shouldn't have been changed

Reply:
It seems that the anchor is no longer in the documentation, so I updated it to point to that new anchor. It seems that there is another anchor, similar to the original: https://developers.google.com/maps/documentation/javascript/get-api-key#specifying-a-client-id-when-loading-the-api